storm-core/src/clj/backtype/storm/stats.clj

```diff
@@ -67,7 +67,7 @@
 (defrecord RollingWindowSet [updater extractor windows all-time])
 
 (defn rolling-window-set [updater merger extractor num-buckets & bucket-sizes]
-  (RollingWindowSet. updater extractor (dofor [s bucket-sizes] (rolling-window updater merger extractor s num-buckets)) nil)
+  (RollingWindowSet. updater extractor (dofor [s bucket-sizes] (rolling-window updater merger extractor num-buckets s)) nil)
```

Comment:
Oh, never mind, I should be looking at #433.

@BuDongDong , thanks for the update.  Would you close this pull request?